configuration.md

```diff
@@ -245,6 +254,18 @@ The supported configuration options:
 
 * `mode` - one of `userspace`, `iptables` (default) or `ipvs` (experimental)
 
+### `pod_security_policy`
+
+Specify the default [pod security policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/). Built-in policies are:
+
+- `00-pharos-privileged` - no restrictions (default)
+- `99-pharos-restricted` - restricted policy
+
+```yaml
+pod_security_policy:
+  default_policy: "00-pharos-privileged"
+```
+
```

Comment:
Should we open up a bit what those default policies mean? Maybe at some summary level like: With restricted policy you cannot run anything on host namespace...

Reply:
Need to also mention that all system level things use `00-pharos-privileged` by default

Reply:
Added a bit more info about policies.